docs/user/tutorials/01-20-set-up-tls-gateway.md

```diff
@@ -1,42 +1,67 @@
-# Set Up a TLS Gateway
+# Set Up a TLS Gateway in Simple Mode
 
 This tutorial shows how to set up a TLS Gateway in simple mode.
 
 ## Prerequisites
 
-* Deploy [a sample HTTPBin Service](./01-00-create-workload.md).
-* Set up [your custom domain](./01-10-setup-custom-domain-for-workload.md) and export the following values as environment variables:
-
-  ```bash
-  export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
-  export GATEWAY=$NAMESPACE/httpbin-gateway
-  ```
+* [Deploy a sample HTTPBin Service](./01-00-create-workload.md).
+* [Set up your custom domain](./01-10-setup-custom-domain-for-workload.md).
    
+## Steps
+
+
+<!-- tabs:start -->
+#### **Kyma Dashboard**
+
+1. Go to **Istio > Gateways** and select **Create Gateway**. 
+2. Switch to the `Advanced` tab and provide the following configuration details:
+    - **Name**: `httpbin-gateway`
+    - In the `Selectors` section, add the following selector: 
```

Comment:
The selector for istio is already predefined, do we want to keep it in this guide?

Reply:
I agree, it's not needed